src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/MSBuildRestoreUtility.cs

```diff
@@ -914,10 +914,21 @@ private static bool IsPropertyTrue(IMSBuildItem item, string propertyName)
         /// <param name="logger">Logger used to display warnings and errors.</param>
         public static Task ReplayWarningsAndErrorsAsync(LockFile lockFile, ILogger logger)
```

Comment:
Is this still called in other places?
Mark as deprecated otherwise. 

Even if it is actually called in other places, maybe we should change it to the more generic method and mark as deprecated either way :)

Reply:
I was told to leave it there so as not to make a breaking API change

Reply:
I'm not suggesting to delete, just mark it obsolete.